package.json

```diff
@@ -15,6 +15,8 @@
   "dependencies": {
     "@apollo/react-hooks": "^3.1.3",
     "@apollo/react-ssr": "^3.1.3",
+    "@mdx-js/loader": "^1.5.1",
+    "@next/mdx": "^9.1.4",
```

Comment:
I don't believe this branch is using anything from MDX. Should mdx-js/loader and next/mdx be included here?